### DISCUSSION TITLE
添加功能：账户注销

### DISCUSSION THREAD

**Discussion by lll123github** [e-dialect/hinghwa-dict-backend#259] [Ideas]
还没有想清楚具体的操作，但是既然允许用户注册，就应当允许注销

**Comment by charlie0129** [comment]
你觉得注销要做哪些操作，怎么样才叫注销？

**Comment by Norton-Lin** [comment]
从数据库内彻底删除这个用户的信息？ 不过这样会牵扯到很多模块，比如这个用户提交的语料评论之类的，这些东西在用户彻底删除后要如何处理

**Comment by charlie0129** [comment]
对，这样很麻烦。所以你会发现各大平台不会提供注销的功能的，就算注销了，发表的东西也不会清楚

**Comment by lll123github** [comment]
应该B 站和QQ可以提供注销功能
